ui-frontend/packages/catalog-ui-search/src/main/webapp/component/app/app.tsx

```diff
@@ -529,7 +529,7 @@ const App = ({
                             className={`group-hover:opacity-100 opacity-25 relative py-3 hover:opacity-100 focus-visible:opacity-100 transition-opacity`}
                             Icon={PersonIcon}
                             iconPosition="start"
-                            expandedText={userInstance.getUserName()}
+                            expandedText="user-profile"
```

Comment:
Do we mean to have this change?  This will make the expanded menu show 'user-profile' for each user rather than their username.

Reply:
@georgeshehata seems like a misplacement